src/Microsoft.Azure.WebJobs.Logging/Entities/ContainerActiveEntity.cs

```diff
@@ -78,7 +78,7 @@ public static async Task<ContainerActiveEntity> LookupAsync(CloudTable table, lo
                 TableResult retrievedResult = await table.ExecuteAsync(retrieveOperation);
                 result = (ContainerActiveEntity)retrievedResult.Result;                
             }
-            catch (StorageException e)
+            catch (Cosmos.Table.StorageException e)
```

Comment:
This seems like a subtle change. We had a test that caught this, I assume?

Reply:
Yeah, this is probably the most annoying thing about the lack of unification between tables and the rest of storage. I know I had a few test failures that helped me find these, and I did a review of places we were catching StorageException to check for similar cases that might need fixing.